src/modules/ui/thirdparty/action.ts

```diff
@@ -1,8 +1,10 @@
 import { action } from 'typesafe-actions'
+import { ThirdPartyAction } from './types'
 
-// TP Approval Flow
-export const APPROVAL_FLOW_UPDATE_PROGRESS = '[Update progress] Approval flow '
+// Update TP Action progress
+export const THIRD_PARTY_ACTION_UPDATE_PROGRESS = '[Update progress] Third Party Action'
 
-export const updateApprovalFlowProgress = (progress: number) => action(APPROVAL_FLOW_UPDATE_PROGRESS, { progress })
+export const updateThirdPartyActionProgress = (progress: number, tpAction: ThirdPartyAction) =>
+  action(THIRD_PARTY_ACTION_UPDATE_PROGRESS, { progress, action: tpAction })
```

Comment:
Just to simplify it hahaha
```suggestion
export const updateThirdPartyActionProgress = (progress: number, action: ThirdPartyAction) =>
  action(THIRD_PARTY_ACTION_UPDATE_PROGRESS, { progress, action })
```

Reply:
I would love to, but it conflicts with the `action` fall from the next line 😅, that's why I had to call it with a different name
<img width="740" alt="image" src="https://user-images.githubusercontent.com/8763687/178707510-e7a79326-6295-4234-ada6-bb14b539afe7.png">